src/context/GameContext.js

```diff
@@ -0,0 +1,116 @@
+import { createContext, useState } from 'react';
+
+const GameContext = createContext();
+
+const GameProvider = ({ children }) => {
+  const [currentPlayer, setCurrentPlayer] = useState('');
```

Comment:
You should initialize this to X or O, other-wise you have to reset your game to start playing.